From ada3e13baaf52c720d2522d0bbddadc7632e04df Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 3 May 2024 17:12:42 +0200
Subject: [PATCH 0819/1695] dts: stm32: add a default backup register
 configuration on stm32mp13 boards

The "st,backup-zones" property is mandatory, add a default configuration
at SoC level by moving the one at board level. It can be superseded by
a board configuration.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 81acc44d9c983ffedd22e899990b1c89663b171d dts: stm32: add a default backup register configuration on stm32mp13 boards]
Change-Id: I956b68422adb5fb9d8ca0536184d306bb7472024
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/378283
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/378785
---
 core/arch/arm/dts/stm32mp131.dtsi    | 1 +
 core/arch/arm/dts/stm32mp135f-dk.dts | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp131.dtsi b/core/arch/arm/dts/stm32mp131.dtsi
index a72a75dbe..657b3c24e 100644
--- a/core/arch/arm/dts/stm32mp131.dtsi
+++ b/core/arch/arm/dts/stm32mp131.dtsi
@@ -424,6 +424,7 @@
 			interrupts = <GIC_SPI 128 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc RTCAPB>;
 			ranges;
+			st,backup-zones = <10 5 17>;
 			status = "disabled";
 
 			nvram: nvram@5c00a100 {
diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index d8e5f0ddf..68af2eb20 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -643,7 +643,6 @@
 
 &tamp {
 	status = "okay";
-	st,backup-zones = <10 5 17>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
 	wakeup-source;
-- 
2.47.2

